src/Types.jl

```diff
@@ -178,7 +178,16 @@ function projectfile_path(env_path::String; strict=false)
 end
 
 function manifestfile_path(env_path::String; strict=false)
-    for name in Base.manifest_names
+    man_names = Base.manifest_names isa Tuple ? Base.manifest_names : Base.manifest_names()
+    if first(man_names) == last(man_names) # indicative of a custom manifest name
+        man_path = joinpath(env_path, first(man_names))
```

Comment:
We can get rid of one of these calls to `first(man_names)`.

```suggestion
    first_man_name = first(man_names)
    if first_man_name == last(man_names) # indicative of a custom manifest name
        man_path = joinpath(env_path, first_man_name)
```